Makefile

```
@@ -25,11 +25,19 @@ export SUBCTL_VERSION
 
 # Targets to make
 
-e2e: E2E_ARGS=cluster1 cluster2
+package-charts: submariner-k8s-broker-$(CHARTS_VERSION).tgz submariner-operator-$(CHARTS_VERSION).tgz
```

Phony targets reduce Make’s ability to calculate the work that needs to be done:

```suggestion
CHART_PACKAGES := submariner-k8s-broker-$(CHARTS_VERSION).tgz submariner-operator-$(CHARTS_VERSION).tgz
```